apps/laboratory/package.json

```diff
@@ -5,7 +5,11 @@
   "scripts": {
     "dev:laboratory": "next dev",
     "build:laboratory": "next build",
-    "lint": "eslint . --ext .js,.jsx,.ts,.tsx"
+    "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
+    "playwright:start": "turbo run build; npm run dev:laboratory",
```

Comment:
Since I don't have `turbo` installed, I can't run `npm run playwright:test` locally. We should make a `npm run turbo-build -> turbo run build` script and call that here.

Reply:
I think I can remove the `turbo run build` from the command. Would that address your concern?

Reply:
You can run `npm run build` instead

Reply:
@arein the script is broken now; you can't run `npm run build` in the lab directory; that script does not exist. `turbo run build` worked because it recurses into the parent directory.

Reply:
You are right actually. For everything to work properly / build packages etc. Set up and commands should be ran from the monorepo root. Similarly something like `npm run laboratory` should be ran from there. Doing otherwise i.e. running from individual packages could have unexpected side effects probably

Reply:
ok fixed